wordpress-mobile/gutenberg-mobile
# Publish android artifacts to s3

This PR updates CircleCI to publish `react-native-aztec` & `react-native-bridge` library artifacts to S3. This is due to [Bintray shutting down their services](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/). Here is the Gutenberg PR that handles these changes: https://github.com/WordPress/gutenberg/pull/30421

**To test:**

https://github.com/wordpress-mobile/WordPress-Android/pull/14385 should be used to test the editor to verify these changes.

**PR submission checklist:**
